.yarnrc.yml

```diff
@@ -1,3 +1,5 @@
+nodeLinker: pnp
```

Comment:
This repository does not use Plug'n'Play.

Reply:
I believe it is based on this [thread](https://github.com/nytimes/react-prosemirror/pull/49#discussion_r1321867957), in some cases though you'll need to explicitly define it.

Reply:
Oh, my confusion! It's using Plug'N'Play but not zero installs.

Reply:
This is actually the default, so you shouldn't need it.

Reply:
I had similar issues ensuring plug'n'play was enabled on my machine when working on the Oak repo, I had to explicitly define it in this repo for the same reason I had to define it in the other one. My version of yarn auto-defaults to non-plug-n-play (for one reason or another).

Reply:
Do you have a global yarn rc somewhere that specifies nodeModules?

Reply:
`yarn config --why` might help.

Reply:
My `~/.yarnrc` is blank unfortunately, so not sure where that default would be coming from.

Reply:
> yarn config --why
huh, I had a floating .yarnrc.yml file in a random folder for some reason that was the default, i removed it so should be fine